.github/workflows/documentation-links.yml

```
@@ -16,6 +16,7 @@ concurrency:
 jobs:
   documentation-links:
     runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.full_name == 'python/peps'
```

Good idea. Here's a bit simpler version that doesn't need repo name, good for copy/pasting into other repos :)

```suggestion
    if: github.event.repository.fork == false
```

---

That's the target repo or the one you are making it from? You do want to make it if you are in a fork, you just don't want to make it if the target is a fork.

---

Though, actually, since this is pull_request_target, I think it won't actually get tested here since it's based on the target branch. So I'm not sure that proves anything. So I'll leave it up to you if you are sure it is the target repo. (I'd have to recreate the PR to retrigger this, as well)

---

You are correct, https://github.com/scientific-python/cookie/pull/598 doesn't trip up even though I made it from a fork after adding this!

---

I added it to my fork: https://github.com/hugovk/peps/commit/c0e21996d1f9ed59df809cf8312021b9aa0ff9db

And my own PR didn't trigger it: https://github.com/hugovk/peps/pull/9

Would you like to open a test one to my fork too?

---

I did at https://github.com/hugovk/peps/pull/10, though fork-to-fork should be extra safe.